### PR TITLE
Task Selection Screen color rework

### PR DESCRIPTION
**Pull request by stefaniamak**
# Additions

## Task Selection Screen background

Created two new assets
- `taskSelectionScreenSkinBox.png`
taskSelectionScreenSkinBox
- `taskSelectionScreenSkinScroll.png`
taskSelectionScreenSkinScroll

Made a new skin and inserted them though that one to the task selection screen. 

My goal with these colors was to achieve a more "table-like" feeling to the background, so the blueprint seems like it sits on top of it.

new task selection screen colors

### CODE REVIEW

File: assets/ui/taskSelectionScreen.ui [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "type": "TaskSelectionScreenLayer",
-    "skin": "MasterOfOreon:plantScreen",
```
Comment (jdrueckert) [inline assets/ui/taskSelectionScreen.ui:3]: Is this used anywhere else anymore or can it be removed?

File: assets/skins/taskSelectionScreen.skin [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            "background": "taskSelectionScreenSkinScroll",
+            "background-border": {
+                "top": 4,
+                "bottom": 4,
+                "left": 4,
+                "right": 4
+            }
+        }
+    }
+}
```
Comment (jdrueckert) [inline assets/skins/taskSelectionScreen.skin]: newline

File: assets/ui/taskSelectionScreen.ui (hunk 1 above)
Comment (stefaniamak) [reply assets/ui/taskSelectionScreen.ui:3]: It is still used at the `plantSelectionScreen.ui`.

If we use the new file at the `plantSelectionScreen.ui`, then we can delete:
* `plantScreen.skin` from `MasterOfOreon\assets\skins`
* `plantScreenSkinBox.png` from `MasterOfOreon\assets\textures`
* `plantScreenSkinScroll.png` from `MasterOfOreon\assets\textures`

Shall I proceed with the above changes?

Comment (stefaniamak) [reply assets/ui/taskSelectionScreen.ui:3]: Actually, I'll have to look more into if the `.png` assets are used somewhere else too.

Comment (jdrueckert) [reply assets/ui/taskSelectionScreen.ui:3]: is the `plantSelectionScreen` still used?

Comment (stefaniamak) [reply assets/ui/taskSelectionScreen.ui:3]: Yes, it is. 

On that note, I want to request a different structure for the lists, which will conclude in removing the `plantSelectionScreen` all togetehr. Could I make a new issue about it? or shall I descuss it over the Discord Channel first?

File: assets/skins/taskSelectionScreen.skin (hunk 2 above)
Comment (stefaniamak) [reply assets/skins/taskSelectionScreen.skin]: The `plantScreen.skin` doesn't seem to have a newline as well, should I add it and commit it to this PR?

File: assets/ui/taskSelectionScreen.ui (hunk 1 above)
Comment (jdrueckert) [reply assets/ui/taskSelectionScreen.ui:3]: Please create an issue for it.
I'll merge this PR as there'll be future work (by whomever) based on your issue that will take care of the `plantSelectionScreen` stuff.

**Review by jdrueckert — APPROVED** [review]

File: assets/ui/taskSelectionScreen.ui (hunk 1 above)
Comment (stefaniamak) [reply assets/ui/taskSelectionScreen.ui:3]: Got it!

**Pull request merged**